app/views/smart_proxies/plugins/_puppet_ca.html.erb

```diff
@@ -0,0 +1,12 @@
+<h3><%= _("Puppet Certificate Authority") %></h3>
+<%= show_feature_version('puppetca') %>
+<div class="col-md-4"><%= _('Hosts managed') %></div>
+<div class="col-md-8">
+  <%= Host::Managed.where(:puppet_ca_proxy_id => @smart_proxy.id).count %>
```

Comment:
I'm missing CA counters here, as I think that if you have pending certs it would be useful to see them here as a number.
this also means we are prefetching the data for these numbers, so maybe reconsider loading this whole cert related info in one ajax?